ocs_ci/ocs/resources/packagemanifest.py

```diff
@@ -57,6 +57,7 @@ def __init__(
             **kwargs,
         )
 
+    @retry(ResourceNotFoundError, tries=100, delay=5, backoff=1)
```

Comment:
Retrying for 8 minutes seems really long to me..  I would say if there is no packagemanifest for more than 1 or 2 minutes it's about opening OCP BZ or as someone from OLM team.  So I would rather set it for something like 

```suggestion
    @retry(ResourceNotFoundError, tries=10, delay=10, backoff=1)
```

Reply:
Changed. In my attempt, it passed just before the 7th attempt so this should be fine.